### ISSUE TITLE
publish the JS as NPM package

### ISSUE THREAD

**Issue by tomasc** [tomasc/simple_form_attachments#21]
@asgerb we need to make this gem (and the Markdown editor one among others) pluggable into latest shiny Rails with webpack support. The JS should be published as NPM package.

**Comment by asgerb** [comment]
This sounds like a good idea. But I will have to read up on the whole thing a bit though, seeing as I've yet to move to this new shiny webpack thingy 😉

**Comment by tomasc** [comment]
Yeah, I am on it now and it's a bit of a headache, esp. when you are pulling JS stuff from engines and gems … But it is definitely worth it.
